test/d2l-content-store.html

```diff
@@ -1,40 +1,30 @@
-<!doctype html>
-<html lang="en">
-	<head>
-		<meta charset="utf-8">
-		<meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1, user-scalable=yes">
-		<title>d2l-content-store tests</title>
-		<script src="/node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
-		<script src="/node_modules/mocha/mocha.js"></script>
-		<script src="/node_modules/chai/chai.js"></script>
-		<script src="/node_modules/@polymer/test-fixture/test-fixture.js"></script>
-		<script src="/node_modules/wct-mocha/wct-mocha.js"></script>
-		<script type="module" src="../d2l-content-store.js"></script>
-	</head>
-	<body>
+<!DOCTYPE html>
+<script src="/node_modules/@polymer/test-fixture/test-fixture.js"></script>
+<script src="/node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
+<script src="/node_modules/chai/chai.js"></script>
```

Comment:
stripping out non-essential parts of the html document. Reduces boilerplate and test case indentation